CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -977,17 +977,18 @@ local function CreateSettingsHub()
 		this.VRShield.Visible = true
 		this:HideShield()
 
-		vrMenuOpened = GuiService.MenuOpened:connect(function()
-			local topbarPanel = Panel3D.Get("Topbar3D")
-			panel.localCF = topbarPanel.localCF * CFrame.Angles(math.rad(-5), 0, 0) * CFrame.new(0, 4, 0) * CFrame.Angles(math.rad(-15), 0, 0)
-			panel:SetVisible(true)
+		vrMenuOpened = this.SettingsShowSignal:connect(function(visible)
+			if visible and this.Visible then
```

Comment:
why do we check this.Visible there? is this if statement equivalent to asking if we are trying to open the settings hub when it is already open?